templates/focus-ios.xliff

```diff
@@ -381,6 +381,42 @@
         <source>Close</source>
         <note>Button label used to close a menu that displays as a popup.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Button.Title">
+        <source>Start browsing</source>
+        <note>Text for a label that indicates the title of button from onboarding screen</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.History.Description">
+        <source>Erase your browsing history, passwords, cookies, and prevent unwanted ads from following you in a simple click!</source>
+        <note>Text for a label that indicates the description of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.History.Title">
+        <source>Your history doesn’t follow you</source>
+        <note>Text for a label that indicates the title of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Incognito.Description">
+        <source>Focus is a dedicated privacy browser with tracking protection and content blocking.</source>
+        <note>Text for a label that indicates the description of incognito section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Incognito.Title">
+        <source>More than just incognito</source>
+        <note>Text for a label that indicates the title of incognito section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Protection.Description">
+        <source>Configure settings so you can decide how much or how little you share.</source>
+        <note>Text for a label that indicates the description of protection section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Protection.Title">
+        <source>Protection at your own discretion</source>
+        <note>Text for a label that indicates the title of protection section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Subtitle">
+        <source>Take your private browsing to the next level.</source>
+        <note>Text for a label that indicates the subtitle for onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Title">
+        <source>Welcome to Firefox %@!</source>
+        <note>Text for a label that indicates the title for onboarding screen. (Focus and Klar)</note>
```

Comment:
Please comment explicitely that the placeholder will be replaced by app name (eg Focus or Klar)

Reply:
corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-32781eb8e0a5f13c0370cf3bf8653e7bb3a02d5269b57edfc8266c10d272c7b3R505-R508